src/main/java/org/apache/commons/lang3/RandomUtils.java

```diff
@@ -77,6 +77,7 @@ public static boolean nextBoolean() {
      * @return the random byte array
      * @throws IllegalArgumentException if {@code count} is negative
      */
+    @SuppressWarnings("index:array.length.negative") // Validate.isTrue => count >= 0
```

Comment:
That means that `count`'s type should be `@NonNegative int`. See https://checkerframework.org/manual/#annotate-normal-behavior